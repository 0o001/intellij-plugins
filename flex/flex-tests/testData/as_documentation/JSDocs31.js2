class Foo {
/**
 * Doc
 * @param x xxx
 * @param y yyy
 * @param z zzz
 * @see Array
 */
function F<caret>oo(x,y,z) {}
}