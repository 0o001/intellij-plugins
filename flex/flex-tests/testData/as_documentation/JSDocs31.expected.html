<PRE><b>Foo.Foo</b>(&nbsp;x,
    &nbsp;y,
    &nbsp;z&nbsp;)
</PRE> Doc
<DL><DT><b>Parameters:</b></DT><DD><code>x</code> -  xxx</DD>
<DD><code>y</code> -  yyy</DD>
<DD><code>z</code> -  zzz</DD>
</DL><DL><DT><b>See also:</b></DT><DD><a href="psi_element://Array">Array</a></DD></DL>